src/Symfony/Bridge/Doctrine/Logger/DbalLogger.php

```diff
@@ -49,12 +49,8 @@ public function startQuery($sql, array $params = null, array $types = null)
             $this->stopwatch->start('doctrine', 'doctrine');
         }
 
-        if (is_array($params)) {
-            $params = $this->normalizeParams($params);
-        }
-
         if (null !== $this->logger) {
-            $this->log($sql, null === $params ? array() : $params);
+            $this->log($sql, null === $params ? array() : $this->normalizeParams($params));
```

Comment:
What I mean is that `$params` can indeed be `null` here but `normalizeParams()` doesn't handle `null`.